package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "lab-dana",
+  "version": "1.0.0",
+  "description": "![CF](https://camo.githubusercontent.com/70edab54bba80edb7493cad3135e9606781cbb6b/687474703a2f2f692e696d6775722e636f6d2f377635415363382e706e67) Lab 13: Single Resource Mongo and Express API ===",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "DEBUG='card*' ./node_modules/mocha/bin/mocha",
+    "start": "DEBUG='card*' nodemon server.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dkulp23/13-mongo_single_resource_api.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/dkulp23/13-mongo_single_resource_api/issues"
+  },
+  "homepage": "https://github.com/dkulp23/13-mongo_single_resource_api#readme",
+  "dependencies": {
+    "bluebird": "^3.4.7",
+    "body-parser": "^1.16.1",
+    "cors": "^2.8.1",
+    "debug": "^2.6.1",
+    "express": "^4.14.1",
+    "http-errors": "^1.6.1",
+    "mongoose": "^4.8.5",
+    "morgan": "^1.8.1"
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "mocha": "^3.2.0",
+    "superagent": "^3.5.0"
```

Comment:
You are missing gulp from your list of dependencies so even though you included the file it will not work since you have not added in gulp.